src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -610,131 +602,138 @@ export const TradeInput = memo(({ isCompact }: TradeInputProps) => {
   return (
     <TradeSlideTransition>
       <WithLazyMount shouldUse={hasUserEnteredAmount} component={GetTradeQuotes} />
-      <MessageOverlay show={isKeplr} title={overlayTitle}>
-        <Flex
-          width='full'
-          justifyContent='center'
-          maxWidth={isCompact || isSmallerThanXl ? '500px' : undefined}
-        >
-          {(isCompact || isSmallerThanXl) && (
-            <Center width='inherit' display={!isCompactQuoteListOpen ? 'none' : undefined}>
-              <QuoteList
-                onBack={handleCloseCompactQuoteList}
+      <WarningAcknowledgement
+        message={`This trade may be unssafe, proceed with caution.`}
```

Comment:
likewise here re translation

Reply:
Actioned in https://github.com/shapeshift/web/pull/6647/commits/05c292f7035995dab341ddf89c2ad7e64e3e383b.